guides/micronaut-aws-secretsmanager-rotation/java/src/main/java/example/micronaut/SecretsManagerRotationStep.java

```diff
@@ -0,0 +1,41 @@
+package example.micronaut;
+
+import io.micronaut.core.annotation.NonNull;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+
+public enum SecretsManagerRotationStep {
+
+    CREATE_SECRET("createSecret"),
+    SET_SECRET("setSecret"),
+    TEST_SECRET("testSecret"),
+    FINISH_SECRET("finishSecret");
+
+    private static final Map<String, SecretsManagerRotationStep> ENUM_MAP;
+
+    static {
+        Map<String, SecretsManagerRotationStep> map = new ConcurrentHashMap<>();
```

Comment:
I doubt you need to worry about thread safety here, a regular HashMap is probably enough